scopes/component/component/component.docs.mdx

```diff
@@ -6,3 +6,43 @@ labels: ['component', 'bit', 'extension', 'aspect']
 import { Component } from '@teambit/component.aspect-docs.component';
 
 <Component />
+
+## Expanding this aspect
+
+Bit allows you to register to slots and expand the component aspect with new capabilities.
+
+### Available slots:
+
+This aspect can be expanded via its available slots, to allow new capabilities to the component page ui.
+
+#### `registerConsumeMethod`:
+
+This method allows you to register new ways to consume a component, and present it in the component page ui. It will add any custom components to the `use-box` in the page's header.
+
+First, require the component aspect to your aspect's code, and use `registerConsumeMethod` to add your custom **consume method**.
+
+`registerConsumeMethod` expects a callback function that is provided with the `ComponentModel` data in it's `args` and returns an object with `component` and a `title` of type `ReactNode` that you would need to supply in order for them to render in the menu.
```

Comment:
@odedre write something about that you can return undefined in title/component to not render it for specific conditions

Reply:
also something about the ordering